### DISCUSSION TITLE
Hydro VS. Razor Pages - Performance

### DISCUSSION THREAD

**Discussion by orawalters** [hydrostack/hydro#42] [Q&A] [answered]
I know the differences in behavior. How does Hydro compare to a normal Razor Pages project in terms of server load? The diffing process sounds computationally expensive.

**Comment by kjeske** [comment, marked as answer]
Hi! The diffing process is done on the client side using the AlpineJs [Morph plugin](https://alpinejs.dev/plugins/morph). So from the server perspective, the overall performance should be the same as rendering regular Razor Pages views.
